.circleci/config.yml

```diff
@@ -101,6 +101,121 @@ jobs:
             chmod +x install_manager.sh
             ./install_manager.sh
 
+  install_cluster:
+    machine: true
+    environment:
+      CONTAINER_NAME: cfy_manager
+      IMAGE_NAME: cfy_manager_image
+      CLUSTER_IMAGE: cfy_manager_image_preinstalled
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Build base container
+          command: |
+            set -eux
+            docker build --tag ${IMAGE_NAME} packaging/docker
+            cp -fr /tmp/workspace/x86_64/*.rpm .circleci/cluster
+            docker build --tag ${CLUSTER_IMAGE} .circleci/cluster
+      - run:
+          name: Prepare name envvars
+          command: |
+            echo "export MANAGER1_NAME=${CONTAINER_NAME}_1" >> $BASH_ENV
+            echo "export MANAGER2_NAME=${CONTAINER_NAME}_2" >> $BASH_ENV
+            echo "export DB_NAME=${CONTAINER_NAME}_db" >> $BASH_ENV
+            echo "export QUEUE_NAME=${CONTAINER_NAME}_queue" >> $BASH_ENV
+      - run:
+          name: Run base container
+          command: |
+            set -eux
+            function run_container {
+              docker run --name $1 -d -v /tmp/workspace:/tmp/workspace:ro -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /run --tmpfs /run/lock --security-opt seccomp:unconfined --cap-add SYS_ADMIN ${CLUSTER_IMAGE}
+            }
+            run_container ${MANAGER1_NAME}
+            run_container ${MANAGER2_NAME}
+            run_container ${QUEUE_NAME}
+            run_container ${DB_NAME}
+            docker exec -d ${MANAGER1_NAME} sh -c "systemctl start sshd"
```

Comment:
I'm guessing this was also for debugging purposes?

Reply:
no, it is also present in the non-cluster case, and we use ssh i think only for the sanity check